site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,112 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Local Development Setup: Counters and Lists
+Before using the examples provided in this document for Counters and Lists, please ensure that you have set up the local development version with `CountsAndLists` enabled. You can do this by running the following command:
+
+```bash
+FEATURE_GATES="CountsAndLists=true" go run cmd/sdk-server/main.go --local
+```
+
+This command starts the local development server, which includes a Counter named `conformanceTestCounter` and a List named `conformanceTestList`.
+
+Additionally, please make sure to set the HTTP port to 9358 as the local server uses it. You can do this by running the following command in a different terminal:
+
+```bash
+export AGONES_SDK_HTTP_PORT=9358
+```
```

Comment:
```suggestion
```

Delete all this please - this page isn't covering the local SDK usage, just the REST API.

Reply:
As a note - we could make a section in https://agones.dev/site/docs/guides/client-sdks/local/ on how to enable feature flags (which we have above), which would be useful! @igooch wdyt?

Reply:
I've added sections for `Enabling Specific Features` and `Setting the HTTP port`.